api/swagger-spec/admissionregistration.k8s.io.json

```diff
@@ -75,7 +75,7 @@
    },
    "v1.GroupVersionForDiscovery": {
     "id": "v1.GroupVersionForDiscovery",
-    "description": "GroupVersion contains the \"group/version\" and \"version\" string of a version. It is made a struct to keep extensibility.",
+    "description": "GroupVersion contains the \"group/version\" and \"version\" string of a version. It is made into a struct to keep extensibility.",
```

Comment:
😅what's the fix?

Reply:
@stepyu Thanks for your PR!

However, like @yue9944882 mentioned, the description is fine even currently.

Closing.
/close